### ISSUE TITLE
Setup CI

### ISSUE THREAD

**Issue by thien-do** [moaijs/moai-vue#6]
CI is kind of necessary now since we want the "vue" repo to be heavily community-based.. maybe we can start this right after #5

**Comment by huyng12** [comment]
Do we need to have a new PR for this? If we only need lint check, I think it could be done in #5. Because us Github Actions config support it currently.

https://github.com/moaijs/moai-vue/blob/7036c3867d18941c89be9e3548e7d890451c7961/.github/workflows/node.js.yml#L15

**Comment by thien-do** [comment]
Yeah maybe same with #5 I'm not sure, depend on the size of the PR :D
